src/Components/ActivationKey/AdditionalRepositoriesCard.js

```diff
@@ -57,11 +49,12 @@ const AdditionalRepositoriesCard = (props) => {
             BaseOS and AppStream, are always enabled and do not need to be
             explicitly added to the activation key.
           </Text>
-          <ButtonWrapper />
+          <AddAdditionalRepositoriesButton
+            onClick={handleEditAdditionalRepositoriesToggle}
+          />
           <AddAdditionalRepositoriesModal
             isOpen={isEditAdditionalRepositoriesModalOpen}
             handleModalToggle={handleEditAdditionalRepositoriesToggle}
-            keyName={activationKey.name}
```

Comment:
The keyName must be passed in - without it you cannot save additional repositories.